benchmark/index.js

```diff
@@ -156,3 +169,84 @@ test('initial indexing', async (t) => {
 
   await ended.promise
 })
+
+const KEY1 = '%Xwdpu9gRe8wl4i0ssKyU24oGYKXW75hjE5VCbEB9bmM=.sha25' // root post
+const KEY2 = '%EpzOw6sOBb4RGtofVD43GnfImoiw6NzEEsraHsNXF1g=.sha25' // contact
+const AUTHOR1 = '@ZngOKXHjrvG+cy7Gjx5pSFunUqcePfmDQQxoUlHFUdU=.ed2551'
+const AUTHOR2 = '@58u/J9+5bOXeYRDCYQ9cJ7kklghIpQFPBYxlhKq1/qs=.ed2551'
+
+const queries = {
+  'key one initial': [and(key(KEY1))],
+
+  'key two': [and(key(KEY2))],
+
+  'key one again': [and(key(KEY1))],
+
+  'latest root posts': [
+    and(type('post'), isRoot(), isPublic()),
+    startFrom(0),
+    paginate(25),
+    descending(),
+  ],
+
+  'latest posts': [
+    and(type('post'), isPublic()),
+    startFrom(0),
+    paginate(25),
+    descending(),
+  ],
+
+  'votes one initial': [and(votesFor(KEY1))],
+
+  'votes one': [and(votesFor(KEY1))],
+
+  'votes again': [and(votesFor(KEY1))],
```

Comment:
@arj03 see above, can we delete one of these? Running it three times doesn't seem necessary

Reply:
Yes